docs/content/tutorials/tutorial-recipe/index.md

```diff
@@ -71,8 +71,8 @@ Developers don't need to specify what cloud resources they're using in their app
    default   Applications.Datastores/redisCaches     bicep                            radius.azurecr.io/recipes/local-dev/rediscaches:latest
    ```
 
-{{< alert title="💡 Dev Recipes" color="info" >}}
-Dev environments are preloaded with [`dev` Recipes]({{< ref "guides/recipes/overview#use-community-dev-recipes" >}}), a set of Recipes that allow you to quickly get up and running with lightweight containerized infrastructure. In This how-to guide, the dev Redis Recipe deploys a lightweight Redis container into your Kubernetes cluster.
+{{< alert title="💡 Local-dev Recipes" color="info" >}}
```

Comment:
```suggestion
{{< alert title="💡 local-dev Recipes" color="info" >}}
```

Reply:
I think for the title it should stay uppercase since it's the first word in the title line but can change if needed